k8s.gcr.io/images/k8s-staging-networking/OWNERS

```diff
@@ -0,0 +1,7 @@
+# See the OWNERS file documentation:
+# https://github.com/kubernetes/kubernetes/blob/master/docs/devel/owners.md
+
+approvers:
+  - bowei
```

Comment:
please remove the leading space, see other OWNERS files.

Reply:
/hold

Reply:
On the list items? Done.

It appears some OWNERS files have them indented and some do not. I may have picked a bad example to follow. Thanks for the review!